src/olympia/editors/models.py

```diff
@@ -734,6 +734,8 @@ def calculate_weight(self):
                 .filter(addon=addon)
                 .filter(rating__lte=3, created__gte=one_year_ago)
                 .count() * 2, 100),
+            'reputation': (
+                max(min(int(addon.reputation or 0) * -100, 300), -300)),
```

Comment:
Can you add a comment for future code archaeologists wondering what these magic numbers are?